http_logs/operations/default.json

```diff
@@ -46,7 +46,8 @@
       "on-conflict": "{{on_conflict | default('update')}}",
       "conflict-probability": {{conflict_probability | default(25)}},
       "recency": {{recency | default(0)}},
-      "corpora": "http_logs"
+      "corpora": "http_logs",
+      "detailed-results": true
```

Comment:
This can add quite some client side overhead due to response deserialisation etc., I know we don't regularly run this as part of our nightly suite, but it is a considerable change from the previous default. Can we add a track param for this?